### PR TITLE
Electric razor and wig added to chameleon kit

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request
Hand mirror added to the chameleon kit. 

## Why It's Good For The Game
Distinctive hair is often a giveaway even with being able to wear a chameleon helmet. Being able to change it would allow someone to blend in more completely. 

## Changelog
:cl:
add: Added an electric razor and wig to the chameleon kit, allowing you to disguise your hair also.
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
all a hand mirror does is give the self aware trait for a few seconds. you'd need to add a razor as well, but a wig may be more fitting

**Comment by VictorVow** [comment]
Fixed, swapped for a razor and wig.

**Comment by zeskorion** [comment]
> Fixed, swapped for a razor and wig.

Hand mirror is necessary for self haircuts on the go (they check self aware). Use normal razor though, not straight razor.

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -471,6 +471,8 @@
 	new /obj/item/radio/headset/chameleon(src)
 	new /obj/item/stamp/chameleon(src)
 	new /obj/item/pda/chameleon(src)
+	new /obj/item/razor/straightrazor(src)
```
Comment (zeskorion) [inline code/game/objects/items/storage/uplink_kits.dm]: please make this a normal razor and include the handmirror, as both are necessary for self haircuts

**Review by zeskorion — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/uplink_kits.dm (hunk 1 above)
Comment (VictorVow) [reply code/game/objects/items/storage/uplink_kits.dm]: Fixed

**Review by zeskorion — APPROVED** [review]

**Pull request merged**